src/abstractdataframe/reshape.jl

```diff
@@ -207,8 +207,8 @@ end
 
 Unstack data frame `df`, i.e. convert it from long to wide format.
 
-Row keys and values from value column will be ordered in the order of their
-appearance in the respective vectors.
+Row keys and values from value column will be ordered in the order produced
+by `groupby`.
```

Comment:
That's not very explicit, you have to read the full `groupby` docstring to find out what's the order. I'd just say that it's undefined.

Reply:
I think we can (and should) make it precise. I have proposed something.